### PR TITLE
Feature os and model config

### PR DESCRIPTION
**Pull request by JC53489**
- Added shortcut for biggles.text so you don't have to go through the command palette, which should speed up text workflows.
- Added a configuration for OS so we can specify a different set of SOX arguments for windows which seems to be necessary. Ideally this would be replaced with automatic detection.
- Added a configuration for the openai chat model name to use. Can choose the generic gpt4 and gpt3.5 models with varying context sizes.

### CODE REVIEW

**Comment by jpallen** [comment]
It looks like `process.platform.startsWith('win')` should be a suitable check without needing a manual config value: https://stackoverflow.com/a/70728636

File: package.json [hunk 1]
```diff
@@ -30,9 +30,14 @@
...
-          "key": "ctrl+m",
-          "mac": "cmd+m"
+        "command": "biggles.voice",
+        "key": "ctrl+m",
+        "mac": "cmd+m"
+      },
+      {
+        "command": "biggles.text",
+        "key": "ctrl+n",
+        "mac": "cmd+n"
```
Comment (jpallen) [inline package.json:40]: This is already a pretty common shortcut for a new file. `cmd/ctrl-h` doesn't seem to a have VSCode default. There are probably others too, I didn't do an exhaustive search!

Comment (jpallen) [reply package.json:40]: Or how about `cmd/ctrl+alt+m` for consistency? I think that's a more common pattern and uses up less prime real estate. The main use of this is voice I think.

Comment (JC53489) [reply package.json:40]: Sounds sensible!

**Pull request merged**